src/Symfony/Component/ErrorHandler/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.1.3
```

Comment:
It's a new feature, so the next minor version would be 5.2.0.

Reply:
Thanks - changed.